src/dotnet/project.json

```diff
@@ -51,7 +51,7 @@
     },
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.1.0-preview1-001086-00"
+      "version": "1.1.0-preview1-001100-00"
     },
     "System.Diagnostics.TraceSource": "4.0.0",
```

Comment:
What about the CoreFX dependencies?  Shouldn't these be updated?  Same goes for any CoreClr dependencies we have.  These should all be updated to the version core-setup is using of the two repos.